Toric/ConvexAnalysis/PolyhedralCone.lean

```diff
@@ -87,6 +103,40 @@ lemma IsPolyhedral.top [hE : FiniteDimensional 𝕜 E] : (⊤ : PointedCone 𝕜
       -- We use our auxiliary statement from above
       exact neg_mem_span_R _ hx
 
+theorem IsPolyhedral.isPolyhedral_span_of_isExtreme {c : PointedCone 𝕜 E}
+    (h : IsPolyhedral c) {s : Set E} (he : IsExtreme 𝕜 c s) :
+    IsPolyhedral (span 𝕜 s) := by
+  replace he' := c.mem_span_inter_of_mem_span_of_isExtreme ?_ he
+  · obtain ⟨g, hg⟩ := isPolyhedral_iff_eq_span c |>.mp h
+    refine ⟨(((g : Set E) ∪ {0}) ∩ s).toFinite.toFinset, ?_⟩
+    apply le_antisymm
+    · rw [span_le]
+      simp only [Set.union_singleton, Set.Finite.coe_toFinset]
+      intro x hx
+      exact subset_span hx.2
+    · rw [span_le]
+      intro x hxs
+      replace he := he' ((g : Set E) ∪ {0}) (hg ▸ subset_span) x (hg ▸ he.1 hxs) hxs
+      simp_all
+  · intro r x hx
+    exact smul_mem_of_isExtreme he x hx r.1 r.2
+
+theorem IsPolyhedral.isPolyhedral_of_isExtreme {c d : PointedCone 𝕜 E}
+    (h : IsPolyhedral c) (he : IsExtreme 𝕜 c (d : Set E)) :
+    IsPolyhedral d := by
+  rw [← Submodule.span_eq (p := d)]
+  exact h.isPolyhedral_span_of_isExtreme he
+
+theorem IsPolyhedral.isPolyhedral_span_of_isExposed [TopologicalSpace 𝕜] [TopologicalSpace E]
+    {c : PointedCone 𝕜 E} (h : IsPolyhedral c) {s : Set E} (he : IsExposed 𝕜 c s) :
+    IsPolyhedral (span 𝕜 s) :=
+  h.isPolyhedral_span_of_isExtreme he.isExtreme
+
+theorem IsPolyhedral.isPolyhedral_of_isExposed [TopologicalSpace 𝕜] [TopologicalSpace E]
+    {c d : PointedCone 𝕜 E} (h : IsPolyhedral c) (he : IsExposed 𝕜 c (d : Set E)) :
+    IsPolyhedral d :=
+  h.isPolyhedral_of_isExtreme he.isExtreme
```

Comment:
Might be a matter of taste, but I don't think these are needed as these trivially follow from the fact that exposed sets are extreme. Also, this and a another one down below are the only places in the PR that assumes `TopologicalSpace` instances, which personally I'd like to avoid (everything convex we do for the toric project shouldn't need a topology).

Reply:
Sounds fair, I removed them. :+1: